README.md

```diff
@@ -1,164 +1,291 @@
-# Tower CLI app
+# Nextflow Tower CLI
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+`tw` is [Tower](https://landing.tower.nf/) on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
 
-## Installation
+Tower (a.k.a Nextflow Tower) is a full-stack application for the management of data pipelines and compute resources. It enables collaborative data analysis at scale, on-premises or in any cloud infrastructure.
 
-From the last Github action at artifacts section, download the binary for your OS. Unzip it, give it execution permissions with `chmod +x ./tw` and move it into a folder that it's in your path (ex: `sudo mv ./tw /usr/local/bin/tw`)
+The Tower CLI (a.k.a Nextflow Tower CLI) interacts with instances of Tower, providing an interface to launch pipelines, manage cloud resources and administer the application directly from the terminal.
 
-## Build binary development version
+![tw](assets/img/tw-screenshot.png)
```

Comment:
Would be great if you could regenerate this image relative to the `rnaseq` pipeline and not `nanoseq` @evanfloden? Be good to be consistent in terms of which pipeline we use as our "gold standard"?